package.json

```
@@ -62,7 +62,7 @@
     "@babel/code-frame": "^7.10.4",
     "@babel/runtime": "^7.12.5",
     "@types/aria-query": "^5.0.1",
-    "aria-query": "5.1.3",
+    "aria-query": "^5.3.0",
```

👋 Hey, sorry for my late "review" but since this isn't merged yet I wanted to give my 2cents.

I do think this should be a breaking change because some of the role are changed, which can result in failing tests after the upgrade.

That's why I also propose to pin the `aria-query` version.

```suggestion
    "aria-query": "5.3.0",
```